src/GovUk.Education.ExploreEducationStatistics.Publisher/Extensions/PublisherExtensions.cs

```diff
@@ -16,8 +15,26 @@ public static class PublisherExtensions
         /// <returns>True if the Release is the latest published version of a Release or is one of the included Releases</returns>
         public static bool IsReleasePublished(this Release release, IEnumerable<Guid> includedReleaseIds = null)
```

Comment:
Be good to understand what `includedReleaseIds` actually is used for - as in, why do we allow an arbitrary set of ids to be passed into this method?  Feels like it could use an explanation in a method doc?

Reply:
That is already in the method doc
`/// <param name="includedReleaseIds">Release id's which are not published yet but are in the process of being published</param>`